input/pagecontent/changes.md

```diff
@@ -15,3 +15,4 @@ This change log documents the significant updates and resolutions implemented fr
   - added Must Support to ServiceRequest.priority [FHIR-46939](https://jira.hl7.org/browse/FHIR-46939)
   - AU eRequesting ServiceRequest profile renamed to be AU eRequesting Diagnostic Request [FHIR-46842](https://jira.hl7.org/browse/FHIR-46842)
 - Renamed AU eRequesting Patient Access actor and CapabilityStatement to be AU eRequesting Patient [FHIR-46802](https://jira.hl7.org/browse/FHIR-46802)
+- Changed the [AU eRequesting Server CapabilityStatement](https://build.fhir.org/ig/hl7au/au-fhir-erequesting/CapabilityStatement-au-erequesting-server.html) create and update interactions on AU eRequesting Diagnostic Request from SHALL to SHOULD [FHIR-47141](https://jira.hl7.org/browse/FHIR-47141)
```

Comment:
We don't normally link to the profile definition, please remove hyperlink from the capability statement. 
Interactions are specified on a resource level, ie ServiceRequest instead of profile AU eRequesting Diagnostic Request.
Suggest changing to something like: "Changed the conformance requirement for ServiceRequest create and update interactions from SHALL SHOULD in AU eRequesting Server CapabilityStatement [FHIR-47141](https://jira.hl7.org/browse/FHIR-47141)", or
"Changed ServiceRequest create and update interactions from SHALL SHOULD in AU eRequesting Server CapabilityStatement [FHIR-47141](https://jira.hl7.org/browse/FHIR-47141)"

Reply:
Done